src/m_misc.c

```diff
@@ -493,14 +493,28 @@ default_t defaults[] = {
     "mouse_sensitivity_horiz",
     (config_t *) &mouseSensitivity_horiz, NULL,
     {10}, {0,UL}, number, ss_none, wad_no,
-    "adjust horizontal (x) mouse sensitivity"
+    "adjust horizontal (x) mouse sensitivity for turning"
   },
 
   { //jff 4/3/98 allow unlimited sensitivity
     "mouse_sensitivity_vert",
     (config_t *) &mouseSensitivity_vert, NULL,
     {10}, {0,UL}, number, ss_none, wad_no,
```

Comment:
I think we should set `mouse_sensitivity_vert` and `mouse_sensitivity_horiz_strafe` to `0` by default. We also have a vertmouse toggle which is disabled by default.